server/repositories/db.ts

```diff
@@ -15,11 +16,14 @@ const connectionOptions: ConnectionOptions = {
   password: String(process.env.DATABASE_PASSWORD),
   database: String(process.env.DATABASE_NAME),
   entities: [Grouping, QuestionGroup, Question],
-  migrationsRun: Boolean(process.env.DATABASE_RUN_MIGRATIONS) || false,
+  migrationsRun: Boolean(process.env.DATABASE_RUN_MIGRATIONS),
   migrations: ['dist/db/migrations/*.js'],
-  extra: {
-    ssl: Boolean(process.env.DATABASE_USE_SSL) || false,
-  },
+  ssl: process.env.DATABASE_USE_SSL
+    ? {
+        rejectUnauthorized: false,
```

Comment:
should this be true? in order to reject unauthorized connections?

Reply:
Yeah, I think it should be - I've updated it to be true